### PR TITLE
IIIF output example

### PR DESCRIPTION
**Pull request by stephenwf**
Example IIIF Output for the timeliner

### CODE REVIEW

**Comment by stephenwf** [comment]
@tomcrane I've made those changed, removed the parent time selector and the top-level selector. Could you double check that that is all valid now?

File: src/components/docs/technical-documentation/iiif-output.mdx [hunk 1]
```diff
@@ -38,7 +54,7 @@ The timeliner aims to produce rich, valid IIIF Presentation 3 manifests as its o
                 "type": "Audio",
                 "format": "audio/mp3"
               },
-              "target": "http://digirati.com/timeliner/manifest/1/c1#t=0,1200"
```
Comment (adam-digirati) [inline src/components/docs/technical-documentation/iiif-output.mdx]: Does this mean that on the bubble editor users can modify the start and the end positions for the whole project?
So the first and the last marker are also draggable on the bubble editor. (https://iiif-timeliner.netlify.com/docs/components-bubble-editor-bubble-editor) Which in terms of audio content may makes sense if the audio file contains silence at the beginning or the end. 

But if this is true, are the ranges relative to the time range selected by the annotation target? 

```json
{
  "@context": [
    "http://digirati.com/ns/timeliner",
    "http://www.w3.org/ns/anno.jsonld",
    "http://iiif.io/api/presentation/3/context.json"
  ],
  "id": "http://digirati.com/timeliner/manifest/1",
  "type": "Manifest",
  "label": {
    "en": [
      "Manifest label"
    ]
  },
  "summary": {
    "en": [
      "Description of manifest"
    ]
  },
  "items": [
    {
      "id": "http://digirati.com/timeliner/manifest/1/c1",
      "type": "Canvas",
      "label": {
        "en": [
          "Canvas label"
        ]
      },
      "summary": {
        "en": [
          "Description of canvas"
        ]
      },
      "duration": 1400,
      "items": [
        {
          "id": "http://digirati.com/timeliner/manifest/1/c1/annoPage/1",
          "type": "AnnotationPage",
          "items": [
            {
              "id": "http://digirati.com/timeliner/manifest/1/c1/anno/1",
              "type": "Annotation",
              "motivation": "painting",
              "body": {
                "id": "http://digirati.com/timeliner/manifest/1/c1/sound.mp3",
                "type": "Audio",
                "format": "audio/mp3"
              },
              "target": "http://digirati.com/timeliner/manifest/1/c1#t=100,1300"
            }
          ]
        }
      ]
    }
  ],
  "structures": [
    {
      "id": "http://digirati.com/timeliner/manifest/1/range/r1",
      "type": "Range",
      "label": {
        "en": [
          "R1"
        ]
      },
      "items": [
        {
          "id": "http://digirati.com/timeliner/manifest/1/range/r2",
          "type": "Range",
          "label": {
            "en": [
              "R2"
            ]
          },
          "items": [
            {
              "id": "http://digirati.com/timeliner/manifest/1/range/r4",
              "type": "Range",
              "label": {
                "en": [
                  "R4"
                ]
              },
              "digirati:backgroundColor": "#009900",
              "items": [
                {
                  "id": "http://digirati.com/timeliner/manifest/1/c1#t=0,400",
                  "type": "Canvas"
                }
              ]
            },
            {
              "id": "http://digirati.com/timeliner/manifest/1/range/r5",
              "type": "Range",
              "label": {
                "en": [
                  "R5"
                ]
              },
              "items": [
                {
                  "id": "http://digirati.com/timeliner/manifest/1/c1#t=400,800",
                  "type": "Canvas"
                }
              ]
            }
          ]
        },
        {
          "id": "http://digirati.com/timeliner/manifest/1/range/r3",
          "type": "Range",
          "label": {
            "en": [
              "R3"
            ]
          },
          "items": [
            {
              "id": "http://digirati.com/timeliner/manifest/1/c1#t=800,1200",
              "type": "Canvas"
            }
          ]
        }
      ]
    }
  ]
}

```

Comment (tomcrane) [reply src/components/docs/technical-documentation/iiif-output.mdx]: You could create an extra parent range to contain all of R1, which is not draggable or editable.

> are the ranges relative to the time range selected by the annotation target?

No, always absolute canvas time.

Comment (adam-digirati) [reply src/components/docs/technical-documentation/iiif-output.mdx]: Just for clarification, is this the correct notation for the scenario described above?
```json
{
  "@context": [
    "http://digirati.com/ns/timeliner",
    "http://www.w3.org/ns/anno.jsonld",
    "http://iiif.io/api/presentation/3/context.json"
  ],
  "id": "http://digirati.com/timeliner/manifest/1",
  "type": "Manifest",
  "label": {
    "en": [
      "Manifest label"
    ]
  },
  "summary": {
    "en": [
      "Description of manifest"
    ]
  },
  "items": [
    {
      "id": "http://digirati.com/timeliner/manifest/1/c1",
      "type": "Canvas",
      "label": {
        "en": [
          "Canvas label"
        ]
      },
      "summary": {
        "en": [
          "Description of canvas"
        ]
      },
      "duration": 1400,
      "items": [
        {
          "id": "http://digirati.com/timeliner/manifest/1/c1/annoPage/1",
          "type": "AnnotationPage",
          "items": [
            {
              "id": "http://digirati.com/timeliner/manifest/1/c1/anno/1",
              "type": "Annotation",
              "motivation": "painting",
              "body": {
                "id": "http://digirati.com/timeliner/manifest/1/c1/sound.mp3",
                "type": "Audio",
                "format": "audio/mp3"
              },
              "target": "http://digirati.com/timeliner/manifest/1/c1#t=0,1400"
            }
          ]
        }
      ]
    }
  ],
  "structures": [
    {
      "id": "http://digirati.com/timeliner/manifest/1/range/r1",
      "type": "Range",
      "label": {
        "en": [
          "R1"
        ]
      },
      "items": [
        {
          "id": "http://digirati.com/timeliner/manifest/1/range/r2",
          "type": "Range",
          "label": {
            "en": [
              "R2"
            ]
          },
          "items": [
            {
              "id": "http://digirati.com/timeliner/manifest/1/range/r4",
              "type": "Range",
              "label": {
                "en": [
                  "R4"
                ]
              },
              "digirati:backgroundColor": "#009900",
              "items": [
                {
                  "id": "http://digirati.com/timeliner/manifest/1/c1#t=100,500",
                  "type": "Canvas"
                }
              ]
            },
            {
              "id": "http://digirati.com/timeliner/manifest/1/range/r5",
              "type": "Range",
              "label": {
                "en": [
                  "R5"
                ]
              },
              "items": [
                {
                  "id": "http://digirati.com/timeliner/manifest/1/c1#t=500,900",
                  "type": "Canvas"
                }
              ]
            }
          ]
        },
        {
          "id": "http://digirati.com/timeliner/manifest/1/range/r3",
          "type": "Range",
          "label": {
            "en": [
              "R3"
            ]
          },
          "items": [
            {
              "id": "http://digirati.com/timeliner/manifest/1/c1#t=900,1300",
              "type": "Canvas"
            }
          ]
        }
      ]
    }
  ]
}
```

**Pull request merged**